eli5/explain.py

```diff
@@ -106,6 +106,14 @@ def explain_prediction(estimator, doc, **kwargs):
 
         This argument may be supported or not, depending on estimator type.
 
+    top_targets : int, optional
+        Number of targets to show. When ``top_targets`` is provided,
+        only specified number of targets with highest scores are shown.
```

Comment:
👍  I think it's a great idea, and negative values seem had to confuse with anything else.